scripts/linters/general_purpose_linter.py

```diff
@@ -243,6 +243,18 @@
             'if you\'re having trouble declaring types.'),
         'excluded_files': (),
         'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(r'\$broadcast'),
+        'message': (
+            'Please do not use $broadcast/$on for propagating events. '
```

Comment:
We still use $on in some part of the codebase. Do we need to remove them (And add a check for that as well?)?

Reply:
No, they're still with `init` and `destroy`, so we can't remove them just yet. We just don't want folks using broadcast/on pairs to propagate events!